gravitee-apim-portal-webui-next/src/components/page/page-markdown/page-markdown.component.scss

```diff
@@ -22,3 +24,55 @@ pre {
   white-space: -moz-pre-wrap;
   word-wrap: break-word;
 }
+
+table {
```

Comment:
this is encapsulated into a shadow dom ? right ? no impact on others component ?

Reply:
yes exactly